packages/liblc3/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+pkgname=liblc3
+pkgver=1.0.1
+pkgrel=1
+pkgdesc='Low Complexity Communication Codec library and tools'
+arch=(aarch64 x86_64)
+url='https://github.com/google/liblc3/'
+license=(Apache)
+groups=(pro-audio)
```

Comment:
We may actually not add any group here since it's not purely a pro-audio package